.github/workflows/ci.yml

```diff
@@ -126,6 +126,39 @@ jobs:
         - linux: linkcheck
           pytest: false
 
+  benchmarks:
```

Comment:
I think it would be nicer if this was it's own workflow file?

Reply:
It doesn't have to be this way, but the desire is to have the benchmarks run only after the `core` build actually succeeds

Reply:
So one other major advantage to moving this to it's own workflow is that we can have it react to the addition of the label. Currently the CI workflow isn't triggered by label add (because it would restart all the CI), but if we make a new workflow we can add that trigger. In my opinion this outweighs the benefit of early-exit if core fails.

Reply:
Perhaps it can be a reusable workflow that `cron` calls when it completes successfully?  I'm still a novice with GitHub actions/workflows